src/main.rs

```diff
@@ -94,6 +94,13 @@ fn print_project(project: &WorkspaceInfo) {
                 rust_binaries.push(binary);
             }
         }
+        for lib in &pkg.cdylibs {
+            eprintln!("    {}", sty.apply_to(format!("[cdylib] {}", lib))); 
```

Comment:
i would add version to this ;)

Reply:
these are nested underneath the packages which specify the version, see the example output in my previous comment.

Reply:
oh i'm dumb ignore me